plugins/colorschemes/everforest.nix

```
@@ -0,0 +1,266 @@
+        [
+          0
+          1
+          2
+        ]
+        ''
+          To use transparent background, set this option to `1`.
+
+          If you want more ui components to be transparent (for example, status line
+          background), set this option to `2`.
+        '';
+
+    dim_inactive_windows = mkEverforestBool 0 ''
+      Dim inactive windows. Only works in Neovim currently.
+
+      When this option is used in conjunction with |g:everforest_show_eob| set to 0,
+      the end of the buffer will only be hidden inside the active window. Inside
+      inactive windows, the end of buffer filler characters will be visible in
+      dimmed symbols. This is due to the way Vim and Neovim handle |hl-EndOfBuffer|.
+    '';
+
+    sign_column_background =
+      helpers.defaultNullOpts.mkEnumFirstDefault
+        [
+          "none"
+          "grey"
+        ]
+        ''
+          By default, the color of sign column background is the same as normal text
+          background, but you can use a grey background by setting this option to `'grey'`.
+        '';
+
+    spell_foreground =
+      helpers.defaultNullOpts.mkEnumFirstDefault
+        [
+          "none"
+          "colored"
+        ]
+        ''
+          By default, this color scheme won't color the foreground of |spell|, instead
+          colored under curls will be used. If you also want to color the foreground,
+          set this option to `'colored'`.
+        '';
+
+    ui_contrast =
+      helpers.defaultNullOpts.mkEnumFirstDefault
+        [
+          "low"
+          "high"
+        ]
+        ''
+          The contrast of line numbers, indent lines, etc.
+        '';
+
+    show_eob = mkEverforestBool 1 ''
+      Whether to show |hl-EndOfBuffer|.
+    '';
+
+    float_style =
+      helpers.defaultNullOpts.mkEnumFirstDefault
+        [
+          "bright"
+          "dim"
+        ]
+        ''
+          Style used to make floating windows stand out from other windows. `'bright'`
+          makes the background of these windows lighter than |hl-Normal|, whereas
+          `'dim'` makes it darker.
+
+          Floating windows include for instance diagnostic pop-ups, scrollable
+          documentation windows from completion engines, overlay windows from
+          installers, etc.
+        '';
+
+    diagnostic_text_highlight = mkEverforestBool 0 ''
+      Some plugins support highlighting error/warning/info/hint texts, by default
+      these texts are only underlined, but you can use this option to also highlight
+      the background of them.
+
+      Currently, the following plugins are supported:
+
+      - neovim's built-in language server client
+      - https://github.com/neoclide/coc.nvim
+      - https://github.com/prabirshrestha/vim-lsp
+      - https://github.com/ycm-core/YouCompleteMe
+      - https://github.com/dense-analysis/ale
+      - https://github.com/neomake/neomake
+      - https://github.com/vim-syntastic/syntastic
```

I'd rather these be `[markdown links](url)`:

```suggestion
      - neovim's built-in language server client
      - [coc.nvim](https://github.com/neoclide/coc.nvim)
      - [vim-lsp](https://github.com/prabirshrestha/vim-lsp)
      - [YouCompleteMe](https://github.com/ycm-core/YouCompleteMe)
      - [ale](https://github.com/dense-analysis/ale)
      - [neomake](https://github.com/neomake/neomake)
      - [syntastic](https://github.com/vim-syntastic/syntastic)
```

Same for links in other option descriptions.